homeassistant/components/torque/sensor.py

```diff
@@ -30,6 +32,16 @@
     {
         vol.Required(CONF_EMAIL): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_PIDS, default={}): vol.Schema(
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler